dotnet/Directory.Build.props

```diff
@@ -31,6 +31,13 @@
 		<NoWarn>NU5105;CS0618;CS8032;CS0618;SYSLIB0021;SYSLIB0023</NoWarn>
 		<IsPackable>true</IsPackable>
 	</PropertyGroup>
+	
+	<PropertyGroup Condition="!$(TargetFramework.StartsWith('net4'))">
+		<DeployDirectoryPlatformBin>\gxnetcore\bin</DeployDirectoryPlatformBin>
```

Comment:
Don't like to have a "\" at the start; if the expected DeployDirectory is empty, assemblies are going to be copied to the root of the machine. Why don't move the "\" to the end of DeployDirectory?

Reply:
But i understand that is easy when defining the parameter DeployDirectory by command-line.

Reply:
Agree. Fixed.
<DeployDirectory> by default is $(MSBuildThisFileDirectory)build
but if the user sets it explicitly to an empty one on the command line, that happens.